### DISCUSSION TITLE
How to add constrained regression options in alamo

### DISCUSSION THREAD

**Discussion by xueno6** [IDAES/idaes-pse#1094] [General]
For example, if I write:
"
from idaes.core.surrogate.alamopy import AlamoTrainer
trainer = AlamoTrainer(input_labels=input_labels,
                           output_labels=output_labels,
                           training_dataframe=df_train)
"
I know how to set, for example, the basic functions :
"trainer.config.expfcns=True"

But for the constrained regression options like 'ZMIN' I did not find where to set it

**Comment by andrewlee94** [comment]
@xueno6 The current implementation of ALAMOpy does not support constrained regression. Only those options listed in the documentation are supported: https://idaes-pse.readthedocs.io/en/latest/explanations/modeling_extensions/surrogate/api/alamopy/index.html

We would be like to support these additional options, but we do not currently have the resources to implement these ourselves.
